src/controllers/incident.controller.js

```diff
@@ -46,6 +46,15 @@ const incidentController = {
         }
         const data = incidentServices.updateRedFlagComment(id, comment);
         return responseHandler(res, 200, 'comment updated successfully', data);
+    },
+
+    FilterRedFlag(req, res, next) {
```

Comment:
Be consistent with naming pattern, use cameCase here

Reply:
> Be consistent with naming pattern, use cameCase here

Ok, I have refractor it.